system-tests.js

```diff
@@ -238,4 +242,28 @@ run('ern platform list')
 run('ern platform plugins list')
 run('ern platform plugins search react-native')
 
-afterAll()
+afterAll() */
+
+// System Tests for API
+console.log(`#### Path to Default schema ${pathToDefaultSchema}`)
+console.log(`#### Path to Default API ${pathToDefaultAPI}`)
+console.log(`#### Path to generated API ${workingDirectoryPath}`)
+run(`ern create-api ${apiName} -p ${apiPkgName} --skipNpmCheck --schemaPath ${pathToDefaultSchema}`)
```

Comment:
can we please adapt the existing api test to work with folder content comparison? To limit the test run time